olp-cpp-sdk-core/src/client/OlpClientSettings.cpp

```diff
@@ -26,6 +27,18 @@ unsigned int DefaultBackdownPolicy(unsigned int milliseconds) {
   return milliseconds;
 }
 
+int FullJitterBackdownPolicy(int initial_backdown_period_msec,
+                             size_t retry_count) {
+  constexpr size_t kMaxRetries = 3;
```

Comment:
Why max retries is only 3 and not what the user-specified?

Reply:
Done - replaced with restriction on wait timeout in `OlpClient.cpp`